requestly-android-okhttp/src/main/res/layout/rq_interceptor_more_info_dialog_layout.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:paddingVertical="20dp"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/rq_interceptor_textview"
+        style="@style/RqInterceptor.TextAppearance.Title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/rq_interceptor_device_id"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/rq_interceptor_textview3"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.08"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_default="percent"
+        app:layout_constraintWidth_percent="0.3" />
+
+    <TextView
+        android:id="@+id/rq_interceptor_DeviceId_TextView"
+        style="@style/RqInterceptor.TextAppearance.Title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/rq_interceptor_excess_lime"
```

Comment:
Why are we using `excess_lime` as default? We should show `Not Initialized` or `null` instead